src/calculateRentalCost.js

```diff
@@ -4,7 +4,19 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const shortTerm = 3;
+  const shortTermDiscount = 20;
+  const longTerm = 7;
+  const longTermDiscount = 50;
+  const price = 40;
+
+  if (days < shortTerm) {
+    return days * price;
+  } else if (days < longTerm) {
+    return days * price - shortTermDiscount;
+  }
+
+  return days * price - longTermDiscount;
```

Comment:
The return statement applies a fixed long-term discount for rentals of 7 days or more. Ensure this logic aligns with the intended discount strategy. If the discount should be applied per day beyond a certain threshold, the calculation needs adjustment.